kubernetes/base/leaderelection/leaderelection.py

```diff
@@ -103,6 +107,15 @@ def renew_loop(self):
 
             # failed to renew, return
             return
+    
+    def release(self):
+        lock_status, old_election_record = self.election_config.lock.get(self.election_config.lock.name,
+                                                                        self.election_config.lock.namespace)
+        if lock_status:
+            logging.info("release lock: {}".format(old_election_record.holder_identity))
+            old_election_record.lease_duration = 1
+            old_election_record.holder_identity = None
```

Comment:
please also set [renew time and acquire time](https://github.com/kubernetes/kubernetes/blob/10066243df4313e28704d3770742f161ceea0112/staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go#L302-L303)

Reply:
done